src/components/20-molecules/file-upload/README.md

```diff
@@ -139,6 +138,10 @@ Checks if the file upload reports a "invalid" state (read-only). You can conside
 
 The function-valued `onChange` callback property is executed as soon as the user removes or adds new files. Adding new files encompasses files added via drag & drop as well as files added via the embedded `<axa-input-file>` dialogue. As its sole parameter a `files` array of Blob objects is passed to the callback, having the Typescript signature `files: Blob[]`.
 
+### onInvalid
+
+The callback property `onInvalid` has the parameter boolean invalid and optional a errorMessage. The callback will be triggered whenever the invalid status changes.
```

Comment:
This is a confusing name, as revealed by your own description. Perhaps onValidityChange, or similar?